s4cmb/tod.py

```diff
@@ -42,8 +42,8 @@ def __init__(self, hardware, scanning_strategy, HealpixFitsMap,
                  array_noise_level=None, array_noise_seed=487587,
                  array_noise_level2=None, array_noise_seed2=56736,
                  nclouds=None, corrlength=None, alpha=None,
-                 f0=None, amp_atm=None,
-                 mapping_perpair=False, mode='standard', store_pointing_matrix_input = False, verbose=False):
+                 f0=None, amp_atm=None, mapping_perpair=False, mode='standard', 
+                 store_pointing_matrix_input = False, verbose=False, perturb_el = False, perturb_az = False, seed_pointing = 0., mu_pointing = 0., sigma_pointing = 13.,perturb_pol_angs=False,seed_pa=0.,pa_mu=0.,pa_sig=0.,pa_diff=False,coherent_pa=False):
```

Comment:
`coherent_pa=False` should be removed as the behaviour can be retrieved setting `pa_sig=0`